.travis.yml

```diff
@@ -29,6 +29,6 @@ deploy:
     develop: org-sagebridge-bridgeserver2-deployment-devdevelop
     uat: org-sagebridge-bridgeserver2-deployment-devstaging
     prod: org-sagebridge-bridgeserver2-deployment-prod
-  zip-file: target/BridgeServer2-2.0.war
+  zip-file: target/BridgeServer2-2.1-SNAPSHOT.war
```

Comment:
are you sure you want a snapshot?  typically we want to set to a released version

Reply:
This is the first time the version is being incremented, and it's not used anywhere else. I want to update itto mark our team taking ownership, but it doesn't matter for the deployment. this travis job will upload the zip file with a hash name (e.g. `travis-f594a6bd734f571c395616256187ab3c1d91c56a-1723507589.zip`) and the version that shows in elastic beanstalk matches the hash (e.g. version `travis-f594a6bd734f571c395616256187ab3c1d91c56a-1723507595`)